### DISCUSSION TITLE
Lock app to area without shortcuts

### DISCUSSION THREAD

**Discussion by omrd** [rxhanson/RectanglePro-Community#343] [Ideas]
Is there a way to lock a specific app to a certain area without having to use a shortcut every time a new window is opened?

Scenario: 

I constantly write new Pages documents for my work. Every time I open a window I have to do a shortcut for it to snap to a custom area. Works like a charm but would love it if it just automatically did that.

**Comment by rxhanson** [comment]
Currently this is not built into Rectangle Pro because from what I can tell it would require polling the windows on the display (I usually try to avoid polling). A lot of people have asked for this, so I might add it in with the understanding that it would likely use polling.

**Comment by omrd** [comment]
I wonder if there's some type of NSAccessibility notification that can be observed to when windows are created.

**Comment by omrd** [comment]
Looks like you're spot on. I sent Apple Dev asking about this and this is their response. 

> Thank you for contacting Apple Developer Technical Support (DTS). We have reviewed your request and have concluded that there is no supported way to achieve the desired functionality given the currently shipping system configurations.
> 
> If you would like for Apple to consider adding support for such features, please submit your suggestion request via Feedback Assistant (https://feedbackassistant.apple.com/). For more information on Feedback Assistant, please visit https://developer.apple.com/bug-reporting.


I will file a Feedback report to maybe have some accessibility option to be able to track windows.
